aiinfra-cluster/variables.tf

```diff
@@ -197,6 +197,18 @@ variable "gke_node_count_per_node_pool" {
   default     = 0
 }
 
+variable "slurm_node_count_static" {
```

Comment:
lets add validation for this.

Reply:
Assuming you mean `> 0`, schedmd checks for `static + dynamic > 0` [here](https://github.com/SchedMD/slurm-gcp/blob/87cece0fcac0c37c4f17d5023014ce28ea3c46d5/terraform/slurm_cluster/modules/slurm_partition/variables.tf#L170) which is a little prettier than a `tobool` validation

Reply:
I am thinking about a validation like this https://github.com/GoogleCloudPlatform/ai-infra-cluster-provisioning/blob/develop/aiinfra-cluster/validation.tf#L26. If the orchestrator type is not `slurm` then this value should not be `> 0`

Reply:
oh that totally makes sense, changed it